docs/src/index.md

```diff
@@ -181,8 +185,20 @@ linear (generic function with 2 methods)
 julia> grads = gradient(() -> sum(linear(x)), Params([W, b]))
 Grads(...)
 
+# Apply gradients to model parameters
 julia> grads[W], grads[b]
 ([0.652543 … 0.683588], [1.0, 1.0])
```

Comment:
Deleting in favor of the more descriptive example below (also note that `grads[W]` does not apply the gradient, it is just how we index `Grads`).

```suggestion
```

Reply:
Cool, thanks for the clarification!

Reply:
This seems much more direct than `[grads[p] for p in [W, b]]`, why remove it?

Also, I'm not sure we should describe this as indexing. It's looking something up in a dictionary, whose keys are the arrays themselves (or their IDs). Of course I know it's `Base.getindex` but that's a detail.

Reply:
You're right. I added a new suggestion that keeps the same clarifications as the other example but the less opaque code from this one.